### ISSUE TITLE
Experiment with training the agents using default inputs and different RL algorithms. Adjust the training parameters and hyperparameters (e.g., learning rate, batch size) to understand how they affect performance .

### ISSUE THREAD

**Issue by K33w3** [K33w3/Project2.1#14]
Document the changes made and how it affects the performance of the agents. Clarify if these different RL algorithms could be implemented in phase 2. Why and why not .
